pkg/informer/informer.go

```diff
@@ -44,6 +101,154 @@ func InformerForResource(res schema.GroupVersionResource) (GenericInformer, erro
 	return i, nil
 }
 
+func isResourceAllowed(cm *v1.ConfigMap, group, kind string, allowedList []AllowedResources, deniedList []DeniedResources) bool {
+
+	// fmt.Println(kind, group)
+	var boolVar bool
+
+	list := []string{"events", "projects", "clusters", "clusterstatuses", "oauthaccesstokens"}
```

Comment:
Please keep the comment from the original code that explains why we exclude these.